data_validation/__main__.py

```diff
@@ -258,12 +261,18 @@ def build_config_managers_from_args(args):
 
     format = args.format if args.format else "table"
 
-    use_random_rows = (
-        None if config_type == consts.SCHEMA_VALIDATION else args.use_random_row
-    )
-    random_row_batch_size = (
-        None if config_type == consts.SCHEMA_VALIDATION else args.random_row_batch_size
-    )
+    # Random row validation is not supported for schema validation and
+    # generate-table-partitions
+    use_random_rows = None
```

Comment:
Instead of doing this, we should just leave the random_rows/random_row_batch_size/threshold as an argument in the partition parser. We should just remove it from documentation so people don't use it.

Reply:
Okay, done